pkg/kubevirt/machine.go

```diff
@@ -169,7 +169,7 @@ func (m *Machine) SupportsCheckingIsBootstrapped() bool {
 
 // IsBootstrapped checks if the VM is bootstrapped with Kubernetes.
 func (m *Machine) IsBootstrapped() bool {
-	if !m.IsReady() {
+	if !m.IsReady() || m.sshKeys == nil {
```

Comment:
This is wrong condition
Need to validate that this is not checked if sshKeys is nil
But if it nil it doesn't say that the machine is not bootstraped

Reply:
yep, good catch! i'll fix this

Reply:
actually, after a second look, this appears to be accurate as it is. We only introspect the guest VM to see if the bootstrapping has completed if ssh keys exist.

We already have a function that is supposed to indicate if a machine can be introspected for bootstrap or not `SupportsCheckingIsBootstrapped`. 

The `|| m.sshKeys == nil` in this function is to simply prevent someone from being able to deference a nil pointer later on in the `IsBootstrapped()` function.